statsforecast/core.py

```diff
@@ -70,22 +70,21 @@ def compute_forecasts(self, h, func, xreg=None, level=None, *args):
                 out[h * i : h * (i + 1)] = res
         return out, keys
 
-    def compute_cv(self, h, test_size, func, input_size=None, *args):
+    def compute_cv(self, h, test_size, func, step_size=1, input_size=None, *args):
         # output of size: (ts, window, h)
-        # assuming step_size = 1 for the moment
-        n_windows = test_size - h + 1
+        if (test_size - h) % step_size:
```

Comment:
Is this condition necessary, would it be better to just skip the remainder and place a warning?
If a user wants to reduce computation and has a weird number of time stamps the condition feels inconvenient.